src/cli/cluster/cluster_manager.js

```diff
@@ -62,10 +62,24 @@ module.exports = class ClusterManager {
     bindAll(this, 'onWatcherAdd', 'onWatcherError', 'onWatcherChange');
 
     if (opts.watch) {
-      this.setupWatching([
-        ...settings.plugins.paths,
-        ...settings.plugins.scanDirs
-      ]);
+      this.setupWatching(
+        [
+          ...settings.plugins.paths,
+          ...settings.plugins.scanDirs,
+        ],
+        [
```

Comment:
do we want to remove the outer array here?  i don't think the ignore param expects a nested array